README.md

```diff
@@ -255,27 +254,27 @@ streaming datasets.
 
 Installation
 ------------
-We recommend using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and conda to install Bokeh. Enter this command at a Bash or Windows command prompt:
+The easiest way to install Bokeh is using the [Anaconda Python distribution](https://www.anaconda.com/what-is-anaconda/) and its included *Conda* package management system. This allows Bokeh and its required dependencies to be installed by entering the following command at a Bash or Windows command prompt:
 
 ```
 conda install bokeh
 ```
 
-This installs Bokeh and all needed dependencies.
-
-To begin using Bokeh or to install using `pip`, follow the [Quickstart](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) documentation.
+The Bokeh [quickstart documentation](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) provides information on installing Bokeh using [`pip`](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#quick-installation). It also includes a [getting started](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#getting-started) guide.
```

Comment:
"pip" wil make a very small visual target for users to click. Also for parallelism I think this phrase should begin with another appositive to mirror the "to install" above. Also, the the "getting started" section is useful regardless of the method of installation. Here is seems like it only applies to the pip instructions. Perhaps:

> To install using `pip`, enter the following command at a Bash or Windows command prompt:
> ```
> pip install bokeh
> ```
>
> For more information, refer to the [installation documentation](https://bokeh.pydata.org/en/latest/docs/installation.html).
> 
> Once Bokeh is installed, check out the [Getting Started](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html#getting-started) section of the [Quickstart](https://bokeh.pydata.org/en/latest/docs/user_guide/quickstart.html) .